### PR TITLE
Add a table to tell the availability of each report in each environment

### PR DESCRIPTION
**Pull request by guochen2010**

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/admin/activity-reports/activity-reports.md [hunk 1]
```diff
@@ -84,53 +84,36 @@ To learn more, see [About admin roles](../add-users/about-admin-roles.md) and [A
...
+|[Forms activity](forms-activity-ww.md)|Yes|Yes|No[^1]|No[^1]|No[^1]|
+|[Dynamics 365 Customer Voice activity](forms-pro-activity-ww.md)|Yes|Yes|N/A[^2]|N/A[^2]|N/A[^2]|
+|[Skype for Business Online activity](/SkypeForBusiness/skype-for-business-online-reporting/activity-report)|Yes|Yes|No[^1]|No[^1]|Yes|
+|[Skype for Business Online conference organized activity](/SkypeForBusiness/skype-for-business-online-reporting/conference-organizer-activity-report)|Yes|Yes|No[^1]|No[^1]|Yes|
+|[Skype for Business Online conference participant activity](/SkypeForBusiness/skype-for-business-online-reporting/conference-participant-activity-report)|Yes|Yes|No[^1]|No[^1]|Yes|
+|[Skype for Business Online peer-to-peer activity](/SkypeForBusiness/skype-for-business-online-reporting/peer-to-peer-activity-report)|Yes|Yes|No[^1]|No[^1]|Yes|
+
+[^1]: The report is in plan to be released in the future. The <a href="https://www.microsoft.com/en-us/microsoft-365/roadmap?filters=" target="_blank">Microsoft 365 Roadmap</a> will be updated before the release.
+[^2]: The service is not available in the environment so no plan to release the report.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/activity-reports/activity-reports.md:116]:
```suggestion
[^1] The report is planned to be released in the future. The <a href="https://www.microsoft.com/en-us/microsoft-365/roadmap?filters=" target="_blank">Microsoft 365 Roadmap</a> will be updated before the release.
[^2] The service is not available in the environment so there is no plan to release the report.
```

File: microsoft-365/admin/activity-reports/browser-usage-report.md [hunk 2]
```diff
@@ -22,7 +22,7 @@ description: "Learn how to get a Microsoft browser usage report using the Micros
 
 # Microsoft 365 Reports in the admin center - Microsoft browser usage
 
-The Microsoft 365 **Reports** dashboard shows you an activity overview across the products in your organization. It enables you to drill into individual product level reports to give you more granular insight about the activities within each product. Check out [the Reports overview topic](activity-reports.md). In the Microsoft browser usage report, you can gain insights on Internet Explorer, Microsoft Edge Legacy, and new Microsoft Edge usage. Usage reporting is based on Microsoft 365 online services accessed by using a Microsoft browser.
+The Microsoft 365 Reports dashboard shows you an activity overview across the products in your organization. It enables you to drill into individual product level reports to give you more granular insight about the activities within each product. Check out [the Reports overview topic](activity-reports.md). In the Microsoft browser usage report, you can gain insights on Internet Explorer, Microsoft Edge Legacy, and new Microsoft Edge usage. Usage reporting is based on Microsoft 365 online services accessed by using a Microsoft browser.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/activity-reports/browser-usage-report.md:25]:
```suggestion
The Microsoft 365 Reports dashboard shows you an activity overview across the products in your organization. It enables you to drill in to individual product level reports to give you more granular insight about the activities within each product. Check out [the Reports overview topic](activity-reports.md). In the Microsoft browser usage report, you can gain insights on Internet Explorer, Microsoft Edge Legacy, and new Microsoft Edge usage. Usage reporting is based on Microsoft 365 online services accessed by using a Microsoft browser.
```

File: microsoft-365/admin/activity-reports/yammer-activity-report-ww.md [hunk 3]
```diff
@@ -24,7 +24,7 @@ description: "Get the Yammer Activity report and know more about the number of u
 
 # Microsoft 365 Reports in the admin center - Yammer activity report
 
-As Microsoft 365 admin, the **Reports** dashboard shows you data on the usage of the products within your organization. Check out [activity reports in the admin center](activity-reports.md). With the **Yammer Activity report**, you can understand the level of engagement of your organization with Yammer by looking at the number of unique users using Yammer to post, like or read a message and the amount of activity generated across the organization. 
+As Microsoft 365 admin, the Reports dashboard shows you data on the usage of the products within your organization. Check out [activity reports in the admin center](activity-reports.md). With the **Yammer Activity report**, you can understand the level of engagement of your organization with Yammer by looking at the number of unique users using Yammer to post, like or read a message and the amount of activity generated across the organization.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/activity-reports/yammer-activity-report-ww.md:27]:
```suggestion
As Microsoft 365 admin, the Reports dashboard shows you data on the usage of the products within your organization. Check out [activity reports in the admin center](activity-reports.md). With the **Yammer Activity report**, you can understand the level of engagement of your organization with Yammer by looking at the number of unique users using Yammer to post, like, or read a message and the amount of activity generated across the organization. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @guochen2010, kindly commit the minor additional suggested changes. Many thanks.

**Pull request merged**